src/search_options.hpp

```diff
@@ -68,6 +68,7 @@ struct LambdaOptions : public SharedOptions
     bool            versionInformationToOutputFile = true;
     size_t          maximumQueryBlockSize = 10;
 
+    bool            seedHalfExact = true;
```

Comment:
Half-exact is now on by default and also overlapping seeds.

Reply:
True